eventmesh-dashboard-common/src/main/java/org/apache/eventmesh/dashboard/common/constant/StoreTypeConstant.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.common.constant;
+
+public class StoreTypeConstant {
+    public static final String STORE_TYPE_TYPE_UNKNOWN = "unknown";
+    public static final String STORE_TYPE_STANDALONE = "standalone";
+    public static final String STORE_TYPE_REDIS = "redis";
+    public static final String STORE_TYPE_MYSQL = "mysql";
+    public static final String STORE_TYPE_ROCKETMQ = "rocketmq4";
+    public static final String STORE_SUBTYPE_ROCKETMQ_BROKER = "rocketmq4-broker";
+    public static final String STORE_SUBTYPE_ROCKETMQ_NAMESERVER = "rocketmq4-nameserver";
```

Comment:
How to combine these three concepts together?

Reply:
These two types belong to the same level, but now `STORE_TYPE_ROCKETMQ` is not used.
 It may be more appropriate to change it to `STORE_TYPE_ROCKETMQ_BROKER` and delete `STORE_TYPE_ROCKETMQ`.